documentation/src/main/asciidoc/topics/con_memory_overhead.adoc

```diff
@@ -0,0 +1,17 @@
+[id='memory-overhead_{context}']
+= Memory overhead
+
+Memory overhead is additional memory that {brandname} uses to store entries.
+When you configure a lifespan for entries, for example, {brandname} adds timestamp metadata that increases memory overhead by a few bytes.
+Likewise {brandname} uses a small amount of memory to track entries in the cache and perform eviction when you bound cache size.
+
+An approximate estimate for memory overhead is 200 bytes per entry in JVM heap memory or 60 bytes per entry in off-heap memory.
+It is impossible to determine a precise amount of memory overhead upfront, however, because the overhead that {brandname} adds per entry depends on several factors.
```

Comment:
Nitpicking, but it sounds like a bait-and-switch to me if we start by mentioning "a few bytes" and then say it's actually 200 bytes.

```suggestion
An approximate estimate for memory overhead is 200 bytes per entry in JVM heap memory or 60 bytes per entry in off-heap memory.
It is impossible to determine a precise amount of memory overhead upfront, however, because the overhead that {brandname} adds per entry depends on several factors.
For example setting an eviction limit makes {brandname} use additional memory to track the recency of entries, and expiration requires one or more timestamps for each entry with lifespan or max-idle set.
```

Reply:
Thanks @danberindei bait and switch is never cool. :wink: